src/test/java/seedu/address/model/volunteer/EventNameContainsKeywordsPredicateTest.java

```diff
@@ -68,10 +68,10 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new VolunteerBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone, email and address, but does not match name
+        // Keywords match phone, and email, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
```

Comment:
might have forgotten to remove `Main` and `Street` here!

Reply:
igi m blind